PetAdoption-iOS/PetAdoptionTransportKit/PFPet.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  PFPet.swift
+//  PetAdoptionTransportKit
+//
+//  Created by Kelii Martin on 1/9/18.
+//  Copyright © 2018 Code For Orlando. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public struct PFPet
+{
```

Comment:
Can we make the data in this class completely immutable?

Reply:
Because some of the data coming from the API are arrays, I wasn't able to declare those with `let`.  Same with the bools; they all come in from the API as an array of JSON objects (this API is really crappy).  The `options` array that comes from the API is what drives the values of those bools.  They all have their default values, and if one of those options is actually passed by the API, we need to change the value to reflect that.